README.md

```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
 List the installed pseudopotential families with the command `aiida-pseudo list`.
 You can then use the name of any family in the command line using the `-F` flag.
 
+## Development
+
+### Test
+```shell
```

Comment:
Maybe preface this with "To run the tests, simply clone and install the package locally with the `[tests]` optional dependencies:"